server/src/modules/task/task.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface TaskDTO {
```

Comment:
I'm not sure I 100% understand the purpose of this particular DTO: I'm guessing it has the same fields as `Task` currently? When would you use this DTO instead of using a regular `Task` object? (For example, this DTO still has ID, so even for `createTask` it's not really any different to receive a `Task` or a `TaskDTO` object.)